main.js

```diff
@@ -10,9 +11,21 @@ const ws = require('./src/system/ws');
 
 const { loadApplication } = require('./src/system/loader.js');
 const APPLICATION_PATH = path.join(process.cwd(), '../NodeJS-Application');
+const LOG_FOLDER_PATH = './log';
+const date = new Date().toISOString().substring(0, 10);
+const filePath = path.join(LOG_FOLDER_PATH, `${date}.log`);
```

Comment:
```suggestion
const date = new Date().toISOString().substring(0, 10);
const LOG_FILE = path.join('./log', `${date}.log`);
```

Reply:
`filePath` is ok naming to be inside class constructor, method or just a function but it is not precise enough to be in file-wide context

Reply:
Done